doc/source/programs/gdal_pansharpen.rst

```diff
@@ -77,6 +78,13 @@ More details can be found in the :ref:`gdal_vrttut_pansharpen` section.
     Select behavior when bands have not the same extent. See
     *SpatialExtentAdjustment* documentation in :ref:`gdal_vrttut_pansharpen`
 
+.. option:: -ms_shift <shiftx> <shifty>
```

Comment:
@rouault is this the correct wording to describe msshift ?

Reply:
I am not sure if the name is logical. Shift makes me think that `-ms_shift 0 0` wouldn't do anything, but is is so that actually `ms_shift 1 1` does not do anything?

Reply:
@jratike80 I'm not sure to understand your comment. For the name of the flag, ms_shift is just replicating the MSShift tags that are already present in the VRT. For the actual values, 0 0 does nothing, 1 1 will offset the ms bands by 1 pixel in the bottom-right direction.

Reply:
I read the comment ` e.g. -ms_shift 1 2 specifies that the top left multispectral pixel corresponds to the top left of the panchromatic pixel of row 2 and column 1.` So I thought that values 0 0  would correspond to row 0 and col 0, thus worth one pixel outside the panchromatic image.

Reply:
@tbonfort How you describe the option would be the logical thing, but looking at the code in GDALPansharpenOperation::ProcessRegion(), my impression is that unfortunately it has currently the reverse convention. And I can't find anything in the doc & tests about those options.
I believe:
- the code should be adjusted to reflect the semantics you propose (hopefully that shouldn't break a lot of people...)
- autotest/gdrivers/vrtpansharpen.py should test the MSShiftX and MSShiftY elements
- doc/source/drivers/raster/vrt.rst should mention them in the section related to pansharpening towards the end
- autotest/pyscripts/test_gdal_pansharpen.py should test the new switch